classes/kernel.oeclass

```diff
@@ -15,10 +15,14 @@
 ##          DEPENDS_KERNEL_INITRAMFS to acquire the initramfs.
 ## @useflag ramdisk_image_compression Compression of the ramdisk image. Set to
 ##          none by default.
-## @useflag kernel_dtb Name of the compiled Device tree source (dts).
-## @useflag kernel_dtc Whether to use the linux device tree compiler (dtc).
+## @useflag kernel_dtb Name of the (already compiled) Device tree binary (dtb)
+##          to be installed into boot directory. kernel_dts makes this
+##          unnecessary.
+## @useflag kernel_dtc Whether to use the device tree compiler (dtc) from the
+##          kernel source tree being built. If not set, dtc from oe-lite recipe
+##          will be used.
 ## @useflag kernel_dtc_flags Flags for the device tree compiler (dtc).
-## @useflag kernel_dtc_source Device tree source (dts).
+## @useflag kernel_dts Name of the Device tree source (dts) to be compiled
```

Comment:
Clearly a more appropriate name :)  But perhaps add "file", so it reads "Name of the Device tree source (dts) file to be compiled".